### PR TITLE
chore(*) improve code coverage

### PR DESCRIPTION
**Pull request by tharun208**
**What this PR does / why we need it**: This PR increases the code coverage.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #

**Special notes for your reviewer**:

**PR Readiness Checklist**:

Complete these before marking the PR as `ready to review`:

- [ ] the `CHANGELOG.md` release notes have been updated to reflect any significant (and particularly user-facing) changes introduced by this PR

### CODE REVIEW

File: pkg/kongstate/upstream_test.go [hunk 1]
```diff
@@ -14,31 +14,31 @@ func TestOverrideUpstream(t *testing.T) {
...
-				Upstream: &kong.Upstream{},
-			},
-			Upstream{
+			inKongIngresss: nil,
+			outUpstream: Upstream{
 				Upstream: kong.Upstream{
 					Name: kong.String("foo.com"),
 				},
 			},
+			annotations: nil,
```
Comment (mflendrich) [inline pkg/kongstate/upstream_test.go:33]:
```suggestion
```

No need to initialize the default value.

**Review by mflendrich — APPROVED** [review]
Looks good! Thank you for your help improving the codebase! :100:

**Comment by tharun208** [comment]
> Looks good! Thank you for your help improving the codebase! 💯

will come with more PR's with improving code coverage.

**Comment by tharun208** [comment]
@mflendrich I see this https://github.com/Kong/kubernetes-ingress-controller/tree/main/railgun/pkg/configsecret pkg is not used in the code. can we remove this?

**Comment by tharun208** [comment]
@mflendrich is it good to merged?

**Review by mflendrich — APPROVED** [review]

**Comment by mflendrich** [comment]
> @mflendrich I see this https://github.com/Kong/kubernetes-ingress-controller/tree/main/railgun/pkg/configsecret pkg is not used in the code. can we remove this?

Of course. I've created a PR: https://github.com/Kong/kubernetes-ingress-controller/pull/1475 based on your suggestion!

**Pull request merged**